src/nectarchain/calibration/NectarGain/SPEfit/NectarGainSPE_singlerun.py

```diff
@@ -4,7 +4,7 @@
 from iminuit import Minuit
 import random
 import astropy.units as u
-from astropy.table import QTable,Column,MaskedColumn
+from astropy.table import Table,QTable,Column,MaskedColumn
```

Comment:
I cannot see where Table is used in the code

Reply:
Indeed it's not I remove